.github/actions/mint-token/action.yml

```diff
@@ -47,7 +47,7 @@ runs:
       with:
         script: |
           try {
-            core.warning('This GitHub action is being deprecated. Please migrate to github.com/abcxyz/github-token-minter/actions/minty.')
+            core.warning('This GitHub action is being deprecated. Please migrate to github.com/abcxyz/github-token-minter/.github/actions/minty.')
```

Comment:
Is there a reason we don't put `action.yml` at the root? Then it just works as "uses: 'github.com/abcxyz/github-token-minter@...'"

Reply:
Can we do that without also needing to move all of the javascript to the root as well? 

I'll play with this when I have a few spare minutes. I like the idea if I can do it in a way that doesn't make a mess.

Reply:
#182